### ISSUE TITLE
Your theme works very well on both Linux and macOS, but it does not work partially on Windows

### ISSUE THREAD

**Issue by gusbemacbe** [programVeins/SteamPunk#1]
Vanakkam @programVeins!

I use temporarily Windows because of WSL, but in the most of my time, I use Linux and macOS (only for compiling and deploying the app for iOS and macOS). The Steampunk theme works perfectly on both Linux and macOS, but does not work on Windows. When I set it as a new theme, it shows the half default VSCode theme and half Steampunk theme. 

VSCode refuses to set your theme:

![image](https://user-images.githubusercontent.com/2257024/150449948-c7564c53-b40a-4a1b-a61b-cc02cbe66d1e.png)

Notice the different colour of VSCode menubar:

![image](https://user-images.githubusercontent.com/2257024/150449875-93fa5ce5-95cd-4863-ba4a-5e55e2bc991b.png)

I am uncertain if some colours shown in Windows'VSCode are correct for you in the Steampunk. You can see the whole screen:

![image](https://user-images.githubusercontent.com/2257024/150450068-02f3b1bd-3b49-40d2-b03a-cc9db91fdb46.png)

Maybe you should notice if there's a small difference in your theme on VSCode's web version:

![image](https://user-images.githubusercontent.com/2257024/150450201-aed6a810-516c-49aa-9333-e6fa88ccb212.png)

Since you are exclusively a macOS user and do not want to install Linux or Windows on your Mac, you can use Parallels (recommended) or VMWare or Virtual Box to test your theme on Linux's and Windows' VSCode to correct the incorrect colours. 

I am a fan of your theme because it is similar to Arcane series' Piltover city colours. 

Pirecilya marrum Piltovarya valttukkal!

**Comment by programVeins** [comment]
Olá @gusbemacbe 

I am super glad and happy to hear you love the theme. I haven't seen the Arcane series but I'm fan of steampunk vibe in general! Super glad you love it!

 As for the Windows specific error, pardon me because I haven't tested it out on the windows machine and thanks for the point on using virtualization. Will see what's up with the theme in Windows when I get free time from work! I have only tested it on macOS and it seemed fine by me. Hence the initial release. Wow it's been almost 2 years since I started / worked on the theme. It was just a simple all-nighter personal project to revamp my personal VSCode and I didn't plan on publishing it initially 😅

Will see what's up with the pesky windows/web bug and see if I can squash it real soon!

Eu aprecio que você tenha se esforçado para traduzir para o tâmil. Muito amor de Chennai 🇮🇳

**Comment by gusbemacbe** [comment]
@programVeins 

For your theme to work totally on all the platforms, I suggest you clone @dsifford's [dracula/visual-studio-code](https://github.com/dracula/visual-studio-code) or @arcticicestudio's [Nord](https://github.com/arcticicestudio/nord-visual-studio-code). They were tested on all platforms and all colours were set correctly. 

Here is a small part with incorrect colours on VSCode's integrated terminal. Compare Steampunk with Dracula and Nord:

Steampunk:

![image](https://user-images.githubusercontent.com/2257024/150512703-fc67666d-f82d-42d5-8669-57a4a6ae16d9.png)

Dracula:

![image](https://user-images.githubusercontent.com/2257024/150512797-22bcf1a0-da88-40bb-adab-6f11d82a22a5.png)

Nord:

![image](https://user-images.githubusercontent.com/2257024/150512875-ead73968-b323-4d0c-9047-5079024c1794.png)

I recommend you to use Oh My Bash and Oh My Posh. 

@romkatv's Powerlevel10k colours are hardcoded; therefore, you have to create a custom Powerlevel10k theme like @jseashell's [dracula/powerlevel10k](https://github.com/dracula/powerlevel10k)

**Comment by romkatv** [comment]
> @romkatv's Powerlevel10k colours are hardcoded; therefore,

Not true. See https://github.com/romkatv/powerlevel10k#how-do-i-change-prompt-colors
